src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -135,6 +160,7 @@ private async Task<bool> IsEnabledAsync<TContext>(string feature, TContext appCo
                         var context = new FeatureFilterEvaluationContext()
                         {
                             FeatureName = feature,
+                            FilterIndex = filterIndex,
```

Comment:
I was thinking it would be relevant for telemetry as well (as we'll mark which filter returned true). But I no longer think that we actually need the index for those changes. I'll go ahead with your suggestion.